### PR TITLE
Do not merge - check CoreDNS image

### PR DESCRIPTION
**Pull request by rajansandeep**
**What this PR does / why we need it**:
This PR checks the Docker image of CoreDNS and make it run against the pre-submit tests.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:
This is just a test/check PR and doesn't require any review.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by rajansandeep** [comment]
@chrisohaver @fturib

**Comment by neolit123** [comment]
@rajansandeep hi, is this related to https://github.com/kubernetes/kubeadm/issues/998 ?

**Comment by rajansandeep** [comment]
@neolit123 No, it's not.
This PR only checks the docker image of CoreDNS.

File: cmd/kubeadm/app/phases/addons/dns/manifests.go [hunk 1]
```diff
@@ -240,7 +240,7 @@ spec:
         effect: NoSchedule
       containers:
       - name: coredns
-        image: {{ .ImageRepository }}/coredns:{{ .Version }}
+        image: coredns/coredns:{{ .Version }}
```
Comment (johnbelamaric) [inline cmd/kubeadm/app/phases/addons/dns/manifests.go]: why this change?
